lib/src/editor/editor_utils.dart

```diff
@@ -426,8 +427,15 @@ class EditorConfig {
 
   /// Aspect ratio of crop rect
   /// default is custom
+  /// user can't change the ratio if this field is set
```

Comment:
```suggestion
  /// Aspect ratio of crop rect
  /// default is custom
  ///
  /// Typically the aspect ratio will not be changed during the editing process,
  /// but it might be relevant with states (e.g. [ExtendedImageState]).
```